setOrgFolderIndex.groovy

```diff
@@ -1,36 +1,41 @@
-//This is for organization folders or multibranch project indexing
-//The script as is just prints out intervals if they exist
-//The function at the bottom can be called to set the interval
+//By default the script just prints out the current indexing intervals for all jobs.
+//The function at the bottom can be called to set the interval to a new value.
+//To call the function, change the two `each` blocks as follows:
+// .each { setInterval(folder) }
 
 import com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger
 import jenkins.model.Jenkins
 import jenkins.branch.OrganizationFolder
 
 println "Organization Items\n-------"
-Jenkins.instance.getAllItems(jenkins.branch.OrganizationFolder.class).each { it.triggers
+Jenkins.instance.getAllItems(jenkins.branch.OrganizationFolder.class).each { folder -> folder.triggers
```

Comment:
```suggestion
Jenkins.get().getAllItems(jenkins.branch.OrganizationFolder.class).each { folder -> folder.triggers
```

`getInstace()` has been deprecated for a long, long time. We might as well change it to the recommended method.